load_data.py

```diff
@@ -0,0 +1,99 @@
+""" Module to multiple files into the database
+
+Module loads multiple models into database from MODELS_DIR of tf and tflite.
+The file naming format should be of the form modelname_subcategory.tflite or
+modelname_subcategory_(SavedModel/FrozenGraph).pb.
+Can only load one type of "is_canonical" models at a time.
```

Comment:
Do remember to change this if you are changing the meaning of this column ('is_canonical')

Reply:
Changed field to "model_type" in accordance to semantics of use.